src/add-to-app/android/project-setup.md

```diff
@@ -322,6 +322,60 @@ dependencyResolutionManagement {
   }
 }
 
+```
+</br>
+
+### Kotlin DSL based Android Project
+
+After `aar` build, follow these steps to add flutter_module in Koltin DSL based Android Projects.
+
+Inlcude Flutter Module as dependecy in Android Project `app/build.gradle`
+<?code-excerpt title="MyApp/app/build.gradle.kts"?>
+```gradle
+android {
+    buildTypes {
+        release {
+          ...
+        }
+        debug {
+          ...
+        }
+        create("profile") {
+            // change with your own signin configs
+            signingConfig = signingConfigs.getByName("debug")
+            initWith(getByName("debug"))
+        }
+}
+dependencies {
+  // ...
+  debugImplementation 'com.example.flutter_module:flutter_debug:1.0'
+  releaseImplementation 'com.example.flutter_module:flutter_release:1.0'
+  "profileImplementation"("com.example.flutter_module:flutter_profile:1.0")
```

Comment:
This does not look right

Reply:
> This does not look right

I tested this on Groovy and Kotlin DSL based projects.

In Groovy Projects `profileImplementation` is available to use.
`profileImplementation("com.example.flutter_module:flutter_profile:1.0")`

In Koltin DSL `profileImplementation` need to be created in Host Project to use profile version. According to resources i was able to find There multiple ways to implement `profileImplementation`. 

1. `add("profileImplementation", "com.example.flutter_module:flutter_profile:1.0")`  OR
2. `"profileImplementation"("com.example.flutter_module:flutter_profile:1.0")`
[Configuring plugins without type-safe accessors](https://docs.gradle.org/current/userguide/kotlin_dsl.html#ex-configuring-plugins-without-type-safe-accessors)

For sample i have created Project to test. `flutter_module` is used in both host projects.
1. Flutter Module [flutter_module](https://github.com/theshivamlko/basic_flutter_module)
3. Kotlin DSL Android Project [UnitConvertorCompose](https://github.com/theshivamlko/Android-Course-Practice/tree/main/UnitConvertorCompose)
4. Groovy Android Project [FlutterAddToAppGroovy](https://github.com/theshivamlko/FlutterAddToAppGroovy)

Reply:
> In Groovy Projects `profileImplementation` is available to use.
> `profileImplementation("com.example.flutter_module:flutter_profile:1.0")`

That's not what is in the PR. Please re-read the delta.

Reply:
@bartekpacia can you weigh in here? Not sure what's expected in terms of having a `profileImplementation` for Kotlin DSL based projects.

Reply:
The line I highlighted is invalid syntax IIUC

Reply:
This strange syntax indeed looks valid ([see here](https://docs.gradle.org/current/userguide/kotlin_dsl.html#artifact_configurations)). I've never built an add-to-app project though and am not familiar with the build system there.